### PR TITLE
Gaultier homepage

### PR DESCRIPTION
**Pull request by Ikaras998**
the new main Views are ready, i prepared place holders for the 3 upcoming events and a random favorite association for when the user is connected. Add the application logo to the icon and main, everything is working properly

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java [hunk 1]
```diff
@@ -13,6 +17,6 @@
     @Test
     public void isAuthenticated() {
         // check not authenticated
-        assertTrue(mActivityRule.getActivity().isAuthenticated());
+        onView(withId(R.id.main_user_fragment)).check(matches(isDisplayed()));
```
Comment (Dahnepfl) [inline app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: Pourquoi tu as supprimé mon test ? C'était pour vérifier si isAuthenticated() retournait la bonne valeur

Il faut laisser.

Ton test tu le mets dans une autre fonction

Comment (Dahnepfl) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: Sinon tu vas baisser le test coverage sur MainActivity.isAuthenticated()...

Comment (Dahnepfl) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: Genre isTheUserLayoutOpen() par exemple

Comment (Ikaras998) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: ah okay je m occuper de ça

Comment (Ikaras998) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: in fact i don't even need to create two methods as both what you did and what i added test the same thing, so i can just readd what you wrote and leave both in the same method

Comment (Ikaras998) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: there change done in commit 0c11e17

Comment (Dahnepfl) [reply app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java]: The two methods don't tests the same thing.

The first one test if isAuthanticated() is working correctly

The otherone test if the fragment layout is the right one

It's not the same thing and this should not be in the same method at my opinion

But that's okay :+1:

**Pull request merged**